pandas/io/parsers.py

```diff
@@ -341,15 +341,16 @@
 widths : list of ints. optional
     A list of field widths which can be used instead of 'colspecs' if
     the intervals are contiguous.
+delimiter: str, default ``\t ``
```

Comment:
Wrap default value in quotes - at a glance it's kind of trick to see that whitespace is included maybe write as `'\t' + ' '`